modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/InsertTableMenuItem.ts

```diff
@@ -80,7 +80,7 @@ const makeComponents = (cells: Array<Array<AlloyComponent>>): Array<AlloySpec> =
 
 const makeLabelText = (row, col) => GuiFactory.text(`${col + 1}x${row + 1}`);
 
-export const renderInsertTableMenuItem = (spec: Menu.FancyMenuItem): ItemTypes.WidgetItemSpec => {
+export const renderInsertTableMenuItem = (spec: Menu.InsertTableMenuItem): ItemTypes.WidgetItemSpec => {
```

Comment:
Up to you if you fix it now or not, but changing the types like this is probably going to cause type errors in `modules/tinymce/src/themes/silver/main/ts/ui/menus/item/build/FancyMenuItem.ts` when we get around to turning on strict types.

Reply:
Yeah I was aware of that, but to fix it'll need some breaking changes in Bridge which is out of scope here. Worth noting it would had also failed previously as well due to the overloading happening.